htdocs/api/v0.0.2/Candidates.php

```diff
@@ -178,7 +184,7 @@ public function createNew($DoB, $edc, $gender, $PSCID)
     {
         $user = \User::singleton();
         return \Candidate::createNew(
-            $user->getCenterID(),
```

Comment:
I'm not sure that using getCenterIDs()[0] is the right thing to do. It should probably return a 500-series error if count($user->getCenterIDs()) is greater than 1, rather than taking an arbitrary site.